data/resources/statsoftwarereview.json

```
@@ -13,5 +13,10 @@
    "title": "Context",
    "text": "Posts related to the project",
    "url": "/tags/statistical-software"
+},
+{
+   "title": "Volunteer",
+   "text": "Volunteering form for statistical software reviewers.",
```

```suggestion
   "text": "Volunteering form for software reviewers including for statistical software.",
```

Not sure how to clarify that the reviewer pool is the same.